src/main/java/com/amazon/dlic/auth/http/jwt/HTTPExtensionJwtAuthenticator.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package com.amazon.dlic.auth.http.jwt;
```

Comment:
What are your thoughts on this package, com.amazon doesn't make much sense to me.  I'd suggest a move this under a `org.opensearch.security.auth` package or child package.

Reply:
Fixed. That makes sense to me too. I was doing this because I was following the class of `HTTPJwtAuthenticator`.